.github/workflows/test.yml

```diff
@@ -14,11 +14,16 @@ jobs:
         go-version: ${{ matrix.go-version }}
     - name: Checkout code
       uses: actions/checkout@v2
-    - name: Test
+    - name: Test klog
       run: |
         go get -t -v ./...
         go test -v -race ./...
+    - name: Test tools
+      run: |
         cd hack/tools && go test -v -race ./...
+    - name: Test examples
```

Comment:
We could use `cd hack/tools && go test -v -race ./... && cd ../..` and keep everything in one step. I find it cleaner to have different steps with different names.

I also tried `(cd  hack/tools && go test -v -race ./...)`. That worked on Linux, but not on Windows because `cd` still had an effect also after the subshell?!